homeassistant/components/amcrest/binary_sensor.py

```diff
@@ -211,13 +211,16 @@ async def _async_update_others(self) -> None:
             log_update_error(_LOGGER, "update", self.name, "binary sensor", error)
             return
 
-        if (event_code := self.entity_description.event_code) is None:
+        if (event_codes := self.entity_description.event_codes) is None or len(
```

Comment:
`not event_codes` is simpler than checking length of the container. All Python containers support truthy check by default to mean empty / not empty.

Reply:
Sounds good, changed.